examples/slice-pizza/datadog-app/src/types/Pizza.ts

```diff
@@ -1,7 +1,9 @@
 export default interface Pizza {
     description: string;
     id: string;
+    image: string;
     name: string;
     price: number;
     quantity: number;
+    amount: number
```

Comment:
I think at one point we assign to `quantity`, but it seems like the API uses `amount` so we sort of end up with both of them being set at some point.